README.md

```diff
@@ -30,25 +30,68 @@ Goal: Validate that the evaluated pipeline configuration with its expressions ar
+* Less issues in production - due to the fast nature of writing and running unit tests, you will write more tests in less time and therefore have a higher test coverage. This means more confidence in new changes, fewer risks in breaking existing features (regression tests), and thus far fewer issues in production.
 
-> Even though Data Factory is UI-driven and writing unit tests might not be in the nature of it. How can you be confident that your changes will work as expected, and existing pipelines will not break, without writing unit tests?
+> Even though Data Factory is UI-driven writing unit tests might not be in the nature of it. How can you be confident that your changes will work as expected, and that existing pipelines will not break, without writing unit tests?
 
 ## Getting started
 
 1. Set up an empty Python project with your favorite testing library
+   In case you want some suggestions about the setup or libraries that can be used. Please use our following setup suggestion: Docs/EnviromnentSetup/Unittest_setup.md
+   [Docs_Setup](https://github.com/data-factory-testing-framework/blob/main/Docs/EnviromnentSetup/Unittest_setup.md)
 2. Install the package using your preferred package manager:
-   * Pip: `pip install data-factory-testing-framework`
-   * Poetry: `poetry add data-factory-testing-framework`
-3. Start writing tests
+    Pip: `pip install data-factory-testing-framework`
+   
+3. Create a Folder in your project and copy the JSON Files with the pipeline definitions locally.
+   In case you need some extra guidance in this affair please check our documentation: Docs/EnviromnentSetup/JSON_pipeline_files.md
+   [Docs Json](https://github.com/data-factory-testing-framework/blob/main/Docs/EnviromnentSetup/JSON_pipeline_files.md)
+  
+4. Start writing tests
+
+## Examples
+
+###  Examples - Loading your JSON file with the Framework and Pytest library:
+
+####  Libraries
+
+```python
+import pytest
+
+from data_factory_testing_framework.state import RunParameterType
+
+from data_factory_testing_framework.state.run_parameter import RunParameter
+
+from data_factory_testing_framework.test_framework import TestFramework, TestFrameworkType
+```
+
+#### Function example
+The piece of the function example will get the JSON file by the name pipeline_fabric_test1 which is the JSON pipeline locally saved in the project using pytest.
+
+
+![image](https://github.com/LiliamLeme/data-factory-testing-framework/assets/62876278/f878dddb-a9ce-40e1-a130-7da51ac67936)
+
+
+
+```python
+def test_pipeline(request: pytest.FixtureRequest) -> None:
+
+  test_framework = TestFramework(
+
+      framework_type=TestFrameworkType.DataFactory, root_folder_path=request.fspath.dirname
+
+    )
+
+  pipeline = test_framework.repository.get_pipeline_by_name("pipeline_fabric_test1")
+
+  ```
 
-## Features - Examples
+### Features - Examples
```

Comment:
Once getting started with examples moved (see prev comment), I think this # can be removed again